src/newLaunch/launchConfig.ts

```diff
@@ -66,16 +65,23 @@ export interface IContactDetails {
   remarks: string
 }
 
+export interface ILaunchDetails {
```

Comment:
Is there any reason the base ILaunchDetails can't also contain fundingTokenInfo, projectTokenInfo and pricePerToken?

Reply:
Oops, ok, I think what I should be asking is:  Is there any reason the base ILaunchDetails can't also contain fundingTokenAddress and pricePerToken?

Reply:
I've missed the `fundingTokenAddress`. Now fixed.
`pricePerToken` is uniqe to Seed.
The `legalDisclaimer` can also be moved here, but I'll do it within the fix to convert the checkbox into an input field in LBP wizard.